source/includes/fundamentals/code-snippets/srv.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+// Connection URI
+const uri = "mongodb+srv://sample-hostname/?maxPoolSize=20&w=majority"
```

Comment:
Nit: Is the intent to match the sample URI included in the image? If so, this may need to be updated.